library/Opus/Model/Plugin/updatedocument_filter.ini

```diff
@@ -42,6 +42,7 @@ cache.Opus_CollectionRole[]=DisplayFrontdoor
 cache.Opus_CollectionRole[]=Visible
 cache.Opus_CollectionRole[]=VisibleBrowsingStart
 cache.Opus_CollectionRole[]=VisibleFrontdoor
+cache.Opus_CollectionRole[]=Language
```

Comment:
Hier wird dafür gesorgt, dass bei Änderungen von 'Language' der Cache gelöscht wird. Um ein Ändern von ServerDateModified zu verhindern, muss oben in der Datei ein Eintrag gemacht werden.

Reply:
Danke für den gründlichen Code Review. 

Auf welchen Cache bezieht sich das?

Müsste dann nicht auch noch
```
cache.Opus_CollectionRole[]=HideEmptyCollections
```
aufgenommen werden? Und warum ist z.B. `DisplayBrowsing` nicht aufgeführt?

Reply:
Den XML-Cache. Bestimmte Felder sind "intern" und ändern nicht die Metadaten des Dokuments. Daher soll ServerDateModified unverändert bleiben. "DisplayBrowsing" gehört zu diesen Feldern. Für die Darstellung sind diese Felder häufig wichtig und der XML-Cache bildet die Grundlage für die Darstellung, insbesondere in der Frontdoor. Beim Browsing funktioniert die Darstellung anders und "DisplayBrowsing" ist effektiv auch ohne, dass der Cache erneuert wird. Die Sammlungen werden direkt aufgelistet und nicht dem XML der Dokumente entnommen.

Reply:
Ich bin mir nicht sicher, dass für das neue Language-Feld ein Update des Caches notwendig ist. Ich vermute nicht, da sich an den Metadaten durch eine Änderung erst einmal nichts ändert.

Das wirft aber eine neue Frage auf, die nichts mehr mit diesem Ticket zu tun hat. Was soll passieren wenn die Sprache (Defaultlocale) für eine Sammlung geändert wird. Der Plan ist die Defaultübersetzung im Name-Feld zu speichern, damit der Rest von OPUS 4 erst einmal so weiter funktionieren kann wie bisher. Bei einem Wechsel des Defaults müssten dann aber eigentlich alle Namen für übersetzte Sammlungen ausgetauscht werden. Ich denke das sollte vermieden werden.